Formula/sysbench.rb

```diff
@@ -4,6 +4,7 @@ class Sysbench < Formula
   url "https://github.com/akopytov/sysbench/archive/1.0.20.tar.gz"
   sha256 "e8ee79b1f399b2d167e6a90de52ccc90e52408f7ade1b9b7135727efe181347f"
   license "GPL-2.0-or-later"
+  revision 2
```

Comment:
```suggestion
  revision 1
```
You'll also need to rebase, since you now have a merge conflict from #79178.

Reply:
All done :)